pkg/client/logs_test.go

```diff
@@ -67,20 +68,20 @@ func TestLateErrors(t *testing.T) {
 
 func TestLogEarlyErrors(t *testing.T) {
 	input := "sonobuoy will help you on your way to greatness"
-	bytestream := make(chan []byte, 1)
-	bytestream <- []byte(input)
-
-	errc := make(chan error, 1)
-	errc <- errors.New("A seriously bad error")
+	bytestream := make(chan []byte)
+	go func() {
```

Comment:
this is the same as writing

bytestream := make(chan []byte, 1)
bytestream <- []byte(input)

Reply:
not sure what you'd like me to do with this comment, can you give me a little more context? do you find this easier to read?